src/omnibase_core/logging/emit.py

```
@@ -443,10 +443,49 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
+        )
+        from omnibase_core.nodes.node_logger.protocols.protocol_context_aware_output_handler import (
+            ProtocolContextAwareOutputHandler,
+        )
+        from omnibase_core.nodes.node_logger.protocols.protocol_smart_log_formatter import (
+            ProtocolSmartLogFormatter,
+        )
+
+        container = ModelONEXContainer()
+        formatter = container.get_service(ProtocolSmartLogFormatter)
+        output_handler = container.get_service(ProtocolContextAwareOutputHandler)
+        return (formatter, output_handler)
```

Ruff (0.14.6)

Consider moving this statement to an `else` block

(TRY300)